server/apis/v1/handler.go

```diff
@@ -118,21 +120,25 @@ func NewHandler(ctx context.Context, dexObj *DexObject, localUsersAuthObject *Lo
 	daemonClientsCache, _ := lru.NewWithEvict[string, daemonclient.DaemonClient](500, func(key string, value daemonclient.DaemonClient) {
 		_ = value.Close()
 	})
+	mvtDaemonClientsCache, _ := lru.NewWithEvict[string, mvtdaemonclient.MonoVertexDaemonClient](500, func(key string, value mvtdaemonclient.MonoVertexDaemonClient) {
+		_ = value.Close()
+	})
 	o := defaultHandlerOptions()
 	for _, opt := range opts {
 		if opt != nil {
 			opt(o)
 		}
 	}
 	return &handler{
-		kubeClient:           kubeClient,
-		metricsClient:        metricsClient,
-		numaflowClient:       numaflowClient,
-		daemonClientsCache:   daemonClientsCache,
-		dexObj:               dexObj,
-		localUsersAuthObject: localUsersAuthObject,
-		healthChecker:        NewHealthChecker(ctx),
-		opts:                 o,
+		kubeClient:            kubeClient,
+		metricsClient:         metricsClient,
+		numaflowClient:        numaflowClient,
+		daemonClientsCache:    daemonClientsCache,
+		mvtDaemonClientsCache: mvtDaemonClientsCache,
+		dexObj:                dexObj,
+		localUsersAuthObject:  localUsersAuthObject,
+		healthChecker:         NewHealthChecker(ctx),
```

Comment:
We have not configured the health checker for monovertex. I believe we can remove for now and we can add back once I add support for this

Reply:
I haven't used healthChecker for mono vertex APIs. Currently, the health status for mono vertex is hardcoded as healthy. We can keep healthChecker and work on expanding it to mono vertex.